api/envoy/service/status/v3/csds.proto

```diff
@@ -53,6 +53,22 @@ enum ConfigStatus {
   ERROR = 4;
 }
 
+// Config status from a client-side view.
+enum ClientSideConfigStatus {
+  // Config status is not available/unknown.
+  CLIENT_UNKNOWN = 0;
+
+  // Client requested the config but hasn't received any config from management
+  // server yet.
+  CLIENT_REQUESTED = 1;
+
+  // Client received the config and replied with ACK.
+  CLIENT_ACKED = 2;
+
+  // Client received the config and replied with NACK.
```

Comment:
Let's explicitly document that the resource content that is returned in this case will be the most recent version accepted, if any, not the version that was NACKed.

Reply:
Documented the expected config dump behavior here and in the PerXdsConfig message.